src/GovUk.Education.ExploreEducationStatistics.Data.Importer/Services/ImporterService.cs

```diff
@@ -86,19 +85,24 @@ private enum Columns
                     Columns.WARD_COLS, new[]{"ward_code", "ward_name"}
                 }
             };
+                
+        private static readonly List<GeographicLevel> IgnoredGeographicLevels = new List<GeographicLevel>
```

Comment:
Should we have a database migration to drop the Institution fields off Location and drop the School and Provider entities?

Reply:
Think leave for now as we'll probably just end up adding again which will be more problematic.